MarvelCharsApp/MarvelCharsApp/Models/Entities/MarvelAPIModel.swift

```diff
@@ -8,18 +8,18 @@
 // This file was generated from JSON Schema using quicktype, do not modify it directly.
 // To parse the JSON, add this file to your project and do:
 //
-//   let marvel = try? newJSONDecoder().decode(Marvel.self, from: jsonData)
+//   let category = try? newJSONDecoder().decode(Category.self, from: jsonData)
 
 import Foundation
 
-// MARK: - MarvelElement
-struct MarvelElement: Codable {
-    let heroes, villains, antiHeroes, aliens: [Alien]
-    let humans: [Alien]
+// MARK: - CategoryElement
+struct CategoryElement: Codable {
```

Comment:
What's the difference between CategoryElement and CategoryModel?

Reply:
By definition, there's no diference between both. Here we follow the recommendations from Wayra and Daniel, for manage separated struct/class for the ApiModel and Entities like "CategoryModel" defined by Fer.